policy/templates/subtemplates/auto/ui-tests.gotmpl

```diff
@@ -68,13 +68,12 @@
           # bring up env, the project name is important
           docker compose -p auto -f pro-ha.yml -f deps_pro-ha.yml -f {{`${{ matrix.db }}`}}.yml --env-file versions.env --profile master-datacenter up --quiet-pull -d
           ./dash-bootstrap.sh http://localhost:3000
-          docker compose -p auto -f pro-ha.yml -f deps_pro-ha.yml -f {{`${{ matrix.db }}`}}.yml --env-file versions.env --profile slave-datacenter up --quiet-pull -d
```

Comment:
Why remove this?

Reply:
In UI tests we do not need mdbc and slave GWs. This is just checking the dashboard UI. Tests are executed very slow on the CI machines (sometimes randomly fails), so maybe without extra components machine will be faster

Reply:
The UI tests templates (this file) and the api-tests template need to be unified to un-duplicate definitions and this adds a variation.
Do you have any evidence to suggest that the tests may run slower without some components? Even a local run on your laptop with and without would help.
I feel that if the tests run slower with the mdcb components that needs to be investigated further rather than being ignored. It is not expected that functionality slows down when mdcb is introduced is it?

Reply:
> The UI tests templates (this file) and the api-tests template need to be unified
We have separate code responsible ([here](https://github.com/TykTechnologies/gromit/tree/master/policy/templates/subtemplates/auto)) for starting env for api-tests and UI tests and **that is OK**, we want to have it that way.
On my local tests are executed at the same speed with and without mdcb, this is visible only when running on slow machines.
Here is comparison from CI
![image](https://github.com/TykTechnologies/gromit/assets/66250826/3e0c0388-3489-4c05-9abc-8b6378e597f7)
Sometimes with mdcb time is also ~17min, but most often is >18min.
And most important -> UI tests are not interacting with mdcb and edge gws (and this is 5 extra docker containers started every time). Even if mdcb would crash UI tests will not detect that. This is just adding complexity, time and consume resources. UI tests are complexity/flakiness hell, simpler pipeline and env would help us contain it

Reply:
So if I understand correctly, this is 30'19" vs 31'25" from one data point. 
We have added larger runners to the org, we can adjust policies so that we run these tests on larger machines. For now, let the environment remain as it is.

Reply:
yes, looks like this discussion in pointless if we will be using larger machines